confgenerator/testdata/valid/default_config/golden_fluent_bit_main.conf

```diff
@@ -148,23 +148,7 @@
     # https://docs.fluentbit.io/manual/pipeline/outputs/stackdriver
     Name     stackdriver
     resource gce_instance
-    Match    ops-agent-collectd
-
-    # https://docs.fluentbit.io/manual/administration/scheduling-and-retries
-    # After 3 retries, a given chunk will be discarded. So bad entries don't accidentally stay around forever.
-    Retry_Limit  3
-
-    # https://docs.fluentbit.io/manual/administration/security
-    # Enable TLS support.
-    tls         On
-    # Do not force certificate validation.
-    tls.verify  Off
-
-[OUTPUT]
-    # https://docs.fluentbit.io/manual/pipeline/outputs/stackdriver
-    Name     stackdriver
-    resource gce_instance
-    Match    syslog
+    Match_Regex    ^(syslog)$
```

Comment:
Can we align these fields? e.g.

```
    Name         stackdriver
    resource     gce_instance
    Match_Regex  ^(syslog)$
```

Reply:
> Looks like Travis CI is failing.

Need to update fluent/conf_test.go

Reply:
Fixed

Reply:
[Format] Looks like the fields are still not aligned?